pom.xml

```diff
@@ -8,8 +8,8 @@
     <modelVersion>4.0.0</modelVersion>
 
     <!-- Project Coordinates -->
-    <groupId>org.apache.spark.scheduler.armada</groupId>
-    <artifactId>armada-cluster-manager_2.13</artifactId>
+    <groupId>io.armadaproject.armada</groupId>
```

Comment:
The Java client uses group `io.armadaproject`, but I'd prefer `io.armadaproject.armada` though.

@dave-gantenbein should the Java client move to `io.armadaproject.armada`, or should we move this Scala client to `io.armadaproject`?

Reply:
@EnricoMi I agree - the Java client should be in `io.armadaproject.armada` namespace.  However, another GR-OSS developer (Sudipto Baral) is working right now on relocating the Java client into the `armada` repository - likely it will be ready for review by end of this week or early next week.  I can ask him to ensure it's using `io.armadaproject.armada`.

Reply:
We've settled with `io.armadaproject` as `armadaproject` is a synonym for `armada`, it is not an umbrella term like an organization.